packages/mux-player/src/template.ts

```diff
@@ -71,9 +71,21 @@ export const content = (props: MuxTemplateProps) => html`
           />`
         : html``}
     </mux-video>
-    <media-loading-indicator slot="centered-chrome" no-auto-hide></media-loading-indicator>
-    ${ChromeRenderer(props)}
-    <mxp-dialog slot="centered-chrome" no-auto-hide open="${props.isDialogOpen}" onclose="${props.onCloseErrorDialog}">
+    <button
+      slot="seek-to-live-button"
+      aria-disabled="${props.inLiveWindow}"
+      onclick="${function (this: HTMLButtonElement, evt: Event) {
+        props.onSeekToLive?.(evt);
+        if (props.paused) {
+          const playRequestEvt = new CustomEvent('mediaplayrequest', { composed: true, bubbles: true });
+          (this as HTMLButtonElement).dispatchEvent(playRequestEvt);
+        }
+      }}"
+      class="mxp-seek-to-live-button"
+    >
+      Live
+    </button>
```

Comment:
this seek to live button I chose to add to the theme via a slot and not have it in the theme itself because of the extra required props like onSeekToLive and inLiveWindow. we could also solve this in the same way as the play button by attaching the click event listener in index.ts but it does separate things more and more. 
mixing imperative with declarative.